ecr-push/action.yml

```diff
@@ -36,11 +40,13 @@ runs:
       shell: bash
       run: |
         digest=${GITHUB_SHA::8}
-        docker tag \
-          ${{ inputs.image }}:${{ inputs.src-tag }} \
-          ${{ steps.login-ecr.outputs.registry }}/${{ inputs.image }}:${digest}
-        docker push \
-          ${{ steps.login-ecr.outputs.registry }}/${{ inputs.image }}:${digest}
+        if [ "${{ inputs.push-digest-tag }}" = "true" ]; then
+          docker tag \
+            ${{ inputs.image }}:${{ inputs.src-tag }} \
+            ${{ steps.login-ecr.outputs.registry }}/${{ inputs.image }}:${digest}
+          docker push \
+            ${{ steps.login-ecr.outputs.registry }}/${{ inputs.image }}:${digest}
+        fi
```

Comment:
I don't know it's really better, but maybe this part could be a separate step.

Reply:
Separated the step